### PR TITLE
8-Solution

### PR DESCRIPTION
**Pull request by araetoro**
I am new to coding so I did get stuck in several areas along the way. I have been working on this for a few days and have learned some new built-in python functions which were really useful! I had a lot of trouble trying to push my work upstream, and ultimately went to my repo on GitHub to add my file, I'm not sure if this works quite right either, Hacktoberfest is definitely a big learning experience for me!

### CODE REVIEW

File: beginner/SumEarnings_Araetoro [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        num_of_nums = input.count(",") +1
+        input_value = list(map(int, input.split(",")))
+        if len(input_value) != num_of_nums:
+            raise ValueError
+        balance = 0
+        for num in input_value:
+            balance += num
+            if balance < 0:
+                balance = 0
+        print(balance)
```
Comment (the-vampiire) [inline beginner/SumEarnings_Araetoro]: hey nice work for your first go!

when you are working on algorithm solutions its a best practice to have your functions return the result. then to call the function and print what it returns to check your outputs.

try refactoring your function to return the solution

**Comment by the-vampiire** [comment]
in terms of using git you can take a look at the README in this repo. if any part of it is still hazy let me know so i can update it

but the overall flow is:
- fork this repo (meaning make a copy of it that you own and have the rights to push to)
- clone your fork using `git clone` on your computer
- do your work and then commit and push back to your forked repo
- open a pull request against this base repo (the one yours is forked from)
- once your PR is reviewed and merged then your code will be added with everyone elses

**Comment by araetoro** [comment]
Thank you for your help. I have updated the code to use return instead of print as recommended. As far as git I was walking through the directions and could get as far as pushing to the upstream branch, but every time I tried my consol said it was an invalid command, I couldn't seem to find the upstream branch. I am not sure if I was naming it incorrectly, but I tried to use the guide provided "git push -u origin issue_number-solution" Not sure where I was going wrong.

**Comment by the-vampiire** [comment]
thats weird. so to be clear you were able to fork and clone it locally?
when you clone a repo it automatically sets the remote for you under the name `origin`

so after you have added and committed your changes it should work to type `git push -u origin your_branch_name`. the branch name has to match exactly as it is named locally. so if you called it 

`araetoro:araetoro-8-solution`

then the command is `git push -u origin araetoro:araetoro-8-solution`

if that doesnt work then try checking your remotes using `git remote --v` you should see the url of your forked repo and the word `origin` next to it

**Pull request merged**